irrd/server/whois/protocol.py

```diff
@@ -35,6 +35,9 @@ def connectionMade(self) -> None:  # noqa: N802
             self.transport.loseConnection()
             return
 
+        # disable nagle
```

Comment:
I think we can remove this comment line, as it merely describes precisely what is happening in the next line.
(I don't seem to have permission to edit your PR myself.)

Reply:
The line mentions TcpNoDelay,  which just so happens setting that also disables nagle.

This was a line from the original author, I actually considered changing it to "disables nagle".
I'm flexible if you think it's still unneeded.

Reply:
OK, let's keep it as is. It's actually not important anyways, the socket handling is quite different in 4.1, so this entire file won't exist in 4.1 anymore.